src/modules.js

```diff
@@ -122,6 +122,10 @@ var LibraryManager = {
       'library_html5.js'
     ];
 
+    if (SUPPORT_CPP_THROW_EXCEPTION) {
```

Comment:
`DISABLE_EXCEPTION_CATCHING` can be used here as well.

Reply:
Unless there is a compelling reason it make sense to have a single option.

Reply:
I would imagine that '-s DISABLE_EXCEPTION_CATCHING=0' at link time would also want to have this effect wouldn't it?     At least in our build system we've been using DISABLE_EXCEPTION_CATCHING as both a link flag and compile flag to enable exceptions in C++ projects, but sine that default is to enable exceptions at link time I guess that will continue to work?